Dockerfile

```diff
@@ -75,9 +78,14 @@ FROM centos:7 AS centos7-build-base
 RUN set -x; yum -y update && \
 		yum -y install git systemd \
 		autoconf libtool libcurl-devel libtool-ltdl-devel openssl-devel yajl-devel \
-		gcc gcc-c++ make bison flex file systemd-devel zlib-devel gtest-devel rpm-build java-11-openjdk-devel \
+		gcc gcc-c++ make bison flex file systemd-devel zlib-devel gtest-devel rpm-build \
 		expect rpm-sign zip
-		ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk/
+		COPY --from=openjdk-install /tmp/OpenJDK17U.tar.gz /tmp/OpenJDK17U.tar.gz
```

Comment:
Just wondering, is there a reason not to use `${OPENJDK_MAJOR_VERSION}` here and below? You'd need to add it as an `ARG` in all containers…

Reply:
i was avoiding adding the arg to all containers, that's all.

Reply:
It's all generated code, anyway. 😁